index.html

```diff
@@ -6,28 +6,35 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
-    <title>Document</title>
+    <title>Portfolio Kazaneza</title>
 </head>
 <body>
 
-    
+    <header>
         <nav class="nav">
-            <label class="logo">KAZANEZA GENTIL</label>
+            <label class="logo"><a href="">KAZANEZA GENTIL</a></label>
             <a href="#">
                 <img class="navImage" src="./Images/ic_menu.svg" alt="Navigation menu">
             </a>
         </nav>
+    </header>   
+        
     
-        <header class="container">
+        <section class="container">
             <div class="headline"> Hey there. <br/> I'm Kazaneza</div>
             <div class="title">I’am a Software Developer</div>
```

Comment:
- [ ] 🛑  as requested by the previous reviewers kindly use `<h1>` tag for your main title: 

hint :
instead of  `<div class="headline"> Hey there. <br/> I'm Kazaneza</div>`  it should be `<h1 class="headline"> Hey there. <br/> I'm Kazaneza</h1>`

- [ ] 🛑 as requested by the previous reviewers please make sure, The subtitle uses a header tag hx `(examples: <h2>,<h3>, etc...): `
hint :
instead of  `<div class="title">I’am a Software Developer</div>`  it should be `<h2 class="title">I’am a Software Developer</h2>`

- [ ] 🛑 as requested by the previous reviewers please also make sure, The paragraph (e.g., "I'm a software ...") is a p tag.
hint:
instead of `<div class="introduction"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, on’t hestiate to contact me.</div>` it should be `<p class="introduction"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, on’t hestiate to contact me.</p>`